### PR TITLE
Change 09500-0151 to use a prefix lookup

### PR DESCRIPTION
**Pull request by haberkornsam**
Use a generic sku prefix for madrid devices to avoid slight differences

### CODE REVIEW

File: devicemaps.json [hunk 1]
```diff
@@ -97,7 +97,7 @@
           }
         ]
       },
-      "90500-0151-G53": {
+      "90500-0151": {
```
Comment (MichaelBaj) [inline devicemaps.json:100]: Is there a way to decorate this JSON file to indicate that this product number reflects the Silicom Madrid series?

**Review by MichaelBaj — APPROVED** [review]

File: devicemaps.json (hunk 1 above)
Comment (BenMatase) [reply devicemaps.json:100]: We could add a field in this obj like `__comment__` or something that we wouldn't use/look at in the HWB, but I don't think we can just add `// comments` as it wouldn't be valid json then.

Comment (agrawalkaushik) [reply devicemaps.json:100]: we can just add a `description` field for something like this.

**Review by agrawalkaushik — APPROVED** [review]

File: devicemaps.json [hunk 2]
```diff
@@ -97,7 +97,8 @@
...
```
Comment (agrawalkaushik) [inline devicemaps.json:100]: @haberkornsam small nit but 

```suggestion
      "90500-0151-": {
```

Just to make a bit more explicity

**Review by BenMatase — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**